cmd/version/version.go

```diff
@@ -61,7 +65,8 @@ func LookupDefaultOCPVersion(releaseStream string) (OCPVersion, error) {
 }
 
 func NewVersionCommand() *cobra.Command {
-	var commitOnly bool
+	var commitOnly, clientOnly bool
+	namespace := "hypershift"
```

Comment:
This assumption might not always be true

Reply:
It's hard-coded into the rest of the repo right now, so not sure how to improve it?